### ISSUE TITLE
[DOCS] Deploy model to SQL Server Machine Learning Services

### ISSUE THREAD

**Issue by parano** [bentoml/BentoML#766]
**Is your feature request related to a problem? Please describe.**

SQL Server Machine Learning Services allow running Python scripts within the database
https://docs.microsoft.com/en-us/sql/machine-learning/sql-server-machine-learning-services?view=sql-server-ver15

Since BentoML saved bundle can be installed directly via `pip`, it should be very straightforward to put a BentoML packaged model into SQL Server for inferencing:
https://docs.microsoft.com/en-us/sql/machine-learning/package-management/install-additional-python-packages-on-sql-server?view=sql-server-ver15#add-the-package-offline

**Describe the solution you'd like**
Add a step-by-step document on how to deploy a model to SQL server with BentoML

**Describe alternatives you've considered**
use something like the following to automate the first step(pip install ...):
```
bentoml sql-server-deploy IrisClassifier:latest  --table-name --column-name
```
table-name, column-name denotes where in the database the BentoML saved bundle will be stored as a BLOB.

**Additional context**

**Comment by PrabhanshuAttri** [comment]
@parano Is this documentation or new feature addition? It isn't clear to me.

**Comment by parano** [comment]
Hi @PrabhanshuAttri, apologize for the confusion, I edited the issue description to make it more clear.  I think adding documentation is gonna be helpful for lots of forks and probably the preferred first step before automating it. let me know if you are interested in looking into this issue, happy to discuss more

**Comment by yashika51** [comment]
Hi @parano, I am interested in working on this. Since this is the first issue I'll be working on for 🍱, I want to focus just on the documentation first and the automation later. 
Which model do we want to deploy to SQL server ml services? 
Should we use the classic Iris classification example or something else?

**Comment by parano** [comment]
hi @yashika51, glad you are interested in taking on this task.  For the documentation, you can go with the Iris Classifier model as an example. When we get to automation later, we'd want to support deploying any python based models to do model inferencing on SQL Server, not just one model or one framework.

**Comment by yashika51** [comment]
Thanks for the reply @parano. I will need to discuss the automation part in detail with you. I'll start the documentation task now 😄.

**Comment by parano** [comment]
@yashika51 sounds great!

**Comment by yubozhao** [comment]
closed with #952
